src/LibExpat.jl

```diff
@@ -265,27 +232,57 @@ end
 # SetBase and GetBase
 
 
+function xp_parse(xph::XPHandle, txt::AbstractString)
+  try
```

Comment:
please stick to 4-space indent like the rest of the code

Reply:
Of course, sorry, bad setting in the editor.